src/generated_proto/google/api/http_pb.js

```diff
@@ -0,0 +1,952 @@
+/**
```

Comment:
Rather than copy this into the repo directly, can we do the same thing that the lightstep-tracer-javascript does. It assumes you have lightstep-common checked out a level above and uses the .proto defined there.

See [here](https://github.com/lightstep/lightstep-tracer-javascript/blob/master/Makefile#L141-L146) for an example.

Reply:
let's talk about this, I'm not sure how exactly I can achieve that